app/src/components/RegisterArtifact.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  RegisterForm,
+  DEFAULT_ERRORS,
+  RegisterOnSubmit,
+  TextFields,
+  ErrorMessages,
+  FilesContext,
+  Document,
+} from './register/RegisterForm';
+import * as React from 'react';
+import Accordion from 'react-bootstrap/Accordion';
+import Card from 'react-bootstrap/Card';
+import FormSubmitButton from './register/FormSubmitButton';
+import { ArtistProvider } from '../providers/ArtistProvider';
+import { ContractProps } from '../helper/eth';
+import ArtistSelection from './register/ArtistSelection';
+import RegisterFields from './register/RegisterFields';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import DropZone from './register/DropZone';
+import ListGroup from 'react-bootstrap/ListGroup';
+import Button from 'react-bootstrap/Button';
+
+const registerArtifact: RegisterOnSubmit = async ({ fields }, _event): Promise<void> => {
+  console.log(fields);
```

Comment:
Is this leftover for debugging? Should this be removed before merging

Reply:
Nope! I forgot to actually submit it to the blockchain. V. Smart